openpype/hosts/maya/plugins/publish/collect_fbx_animation.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from maya import cmds  # noqa
+import pyblish.api
+
+
+class CollectFbxAnimation(pyblish.api.InstancePlugin):
+    """Collect Unreal Skeletal Mesh."""
+
+    order = pyblish.api.CollectorOrder + 0.2
+    label = "Collect Fbx Animation"
+    hosts = ["maya"]
+    families = ["animation"]
+
+    def process(self, instance):
+        skeleton_sets = [
+            i for i in instance
+            if i.lower().endswith("skeletonanim_set")
```

Comment:
I don't think this should ever happen. What is the source of `skeletonamin_set`?

Reply:
because that is the current cause of why **Extract Animation (FBX)** never triggers

Reply:
> I don't think this ever happen. What is the source of `skeletonamin_set`?

The `skeletonamin_set` should be skeletonAnim_SET. So this code is to get the list of selection from skeletonAnim_SET

Reply:
I have figured out why, the anim_skeleton set and skeleton_mesh set have not been added into the loaded rig sets so that's why the animated_skeleton got empty list